vod/mp4/mp4_parser.c

```diff
@@ -533,6 +545,54 @@ mp4_parser_parse_mdhd_atom(atom_info_t* atom_info, metadata_parse_context_t* con
 	return VOD_OK;
 }
 
+static vod_status_t
+mp4_parser_parse_udta_name_atom(atom_info_t* atom_info, metadata_parse_context_t* context)
+{
+	const char* name = (const char*)atom_info->ptr;
+	uint64_t name_len;
+
+	if (name == NULL) {
```

Comment:
1. { convention (note this repeats a few more times)
2. change to atom_info->size == 0 (that's how it's done in other places)